Dockerfiles/email-server/init.sh

```
@@ -1,9 +1,27 @@
 service dovecot start
 service postfix start
 
+start_dir=$PWD
+
 # Check if GENERATE_MESSAGES environment variable is set to 'true'
 if [ "$GENERATE_MESSAGES" = "true" ]; then
+    # Get the apache spamassassin ham dataset from https://spamassassin.apache.org/old/publiccorpus/
+    echo "Fetching samples from spamassassin dataset";
+    mkdir -p /var/lib/spamassassin-dataset;
+    cd /var/lib/spamassassin-dataset;
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)